packages/grafana-ui/src/types/events.ts

```diff
@@ -0,0 +1,35 @@
+import { eventFactory } from '@grafana/data';
+import { DataQueryResponseData, DataQueryError } from '.';
+
+/** Payloads */
+
+export interface PanelChangeViewPayload {
+  fullscreen?: boolean;
+  edit?: boolean;
+  panelId?: number;
+  toggle?: boolean;
+}
+
+export interface MenuElement {
+  text: string;
+  click: string;
+  role?: string;
+  shortcut?: string;
+}
+
+/** Events */
+
+export const clientRefreshed = eventFactory('refresh');
```

Comment:
since we have namespacing of these events now. this could just be refresh to have same name as event string